corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_form_setting.html

```diff
@@ -74,6 +74,20 @@
         </div>
       {% endif %}
     </div>
+    {% if request|toggle_enabled:'FOLLOWUP_FORMS_AS_CASE_LIST_FORM' %}
```

Comment:
It's a little strange that the flag for followup forms also controls relevancy, which could also be useful for regular reg from case list forms, but I'm guessing this is flagged for the sake of not introducing changes for SaaS users?

Reply:
> but I'm guessing this is flagged for the sake of not introducing changes for SaaS users?

Yes, that's right!